homeassistant/components/smlight/strings.json

```diff
@@ -128,6 +128,16 @@
     }
   },
   "issues": {
+    "upgrade_firmware": {
+      "title": "SLZB core firmware update required",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "description": "Your SMLIGHT SLZB-06x device is running an unsupported core firmware version. Please update it to the latest version."
+          }
```

Comment:
wdym unsupported, in that case we should not add the device in the first place

Reply:
I suppose partially supported would be more accurate. We get basic info from the device and the user can update to new firmware, directly in HA from the update entity.  However sensors and other entities wont work until the firmware updates.

Reply:
But if we handle the previous firmware well, we can't do more than just notify the user that they should upgrade? I mean yes, they should upgrade, but we shouldn't use every tool we have to make sure the user does imo

Reply:
Without a deprecation. If the device stops being supported, that might be a reason to raise an issue, but if it just keeps working, there's no difference between an issue and an update entity

Reply:
ok I will remove the issue, and they will just get update notification.